### ISSUE TITLE
Feature Request: Configure SuperGlobals as array

### ISSUE THREAD

**Issue by pableu** [spaze/phpstan-disallowed-calls#110]
This would allow me to make the config significantly easier and shorter: 

```neon
    disallowedSuperglobals:
        -
            superglobal: '$_GET'
            allowIn:
                - lib/*
                - admin/legacy/*
        -
            superglobal: '$_POST'
            allowIn:
                - lib/*
                - admin/legacy/*
        -
            superglobal: '$_REQUEST'
            allowIn:
                - lib/*
                - admin/legacy/*
        -
            superglobal: '$_COOKIE'
            allowIn:
                - lib/*
                - admin/legacy/*
        -
            superglobal: '$_SERVER'
            allowIn:
                - lib/*
                - admin/legacy/*
```

```neon
    disallowedSuperglobals:
        -
            superglobal: ['$_GET', '$_POST', '$_REQUEST', '$_COOKIE', '$_SERVER']
            allowIn:
                - lib/*
                - admin/legacy/*
```

Or is this already possible and I just missed it?

**Comment by spaze** [comment]
Currently this isn't possible but seems useful (not just for `disallowedSuperglobals` but also for other `disallowed*` config keys) so I may add it one day. Feel free to send a pull request :-)

**Comment by spaze** [comment]
Today is the day when I realized it would also save me some config lines so I've added the feature, thanks for the idea! Release soon.

**Comment by pableu** [comment]
Very cool, thank you @spaze!

**Comment by spaze** [comment]
Now released in [2.4.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.4.0).
